templates/download/qualcomm-iot.html

```diff
@@ -0,0 +1,166 @@
+              <a class="p-button"
+                 href="https://artifacts.codelinaro.org/artifactory/qli-ci/flashable-binaries/ubuntu-fw/QLI.1.1-Ver.1.1-ubuntu-nhlos-bins.tar.gz">Download boot firmware</a>
+            </p>
+          </div>
+        </div>
+        <div class="row p-strip u-no-padding--top">
+          <hr class="col-9 p-rule--muted" />
+          <div class="col-3">
+            <h3 class="p-heading--5">Get started</h3>
+          </div>
+          <div class="col-6">
+            <p>
+              Get started with Ubuntu Server 22.04 for <a href="https://www.qualcomm.com/products/internet-of-things/industrial/building-enterprise/qcs6490#overview">QCS6490</a>
+            </p>
+            <hr class="p-rule--muted" />
+            <p>
+              Ubuntu Server 22.04 for QCS6490 <a href="https://docs.qualcomm.com/bundle/resource/topics/80-82645-1/getting-started-with-qualcomm-rb3-gen-2-development-kit_1.html?product=1601111740057201#sub$download-images-and-boot-firmware_1_3">image installation instructions</a>
+            </p>
+            <hr class="p-rule--muted" />
+            <p>
+              Ubuntu Server 22.04 for QCS6490 <a href="TODO">Release Notes (TODO)</a>
+            </p>
+            <hr class="p-rule--muted" />
+            <div class="p-notification--information is-borderless">
+              <div class="p-notification__content">
+                <p class="p-notification__message">
+                  The Ubuntu images are tested on the <a href="https://www.qualcomm.com/products/internet-of-things/industrial/building-enterprise/qcs6490#featuredhardwaresection">Qualcomm® RB3 Gen 2 dev kit</a>
+                </p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-strip is-shallow">
+    <div class="row">
+      <hr class="p-rule" />
+    </div>
+    <div class="row--50-50 p-section">
+      <div class="col">
+        <h2>Are you working on a commercial project?</h2>
+      </div>
+      <div class="col">
+        <p>
+          Canonical partners with silicon vendors, board manufacturers and leading enterprises to shorten time-to-market. If you are deploying Ubuntu on Qualcomm IoT platforms at-scale, <a href="https://ubuntu.com/internet-of-things/qualcomm/contact-us">reach out to Canonical</a> to get access to ongoing bug fixes, critical security patching, long-term support; or to learn more about our solutions for custom board enablement and application development services.
+        </p>
+        <p>
+          For information about using Canonical’s Ubuntu images on a commercial project, <a href="https://ubuntu.com/legal/intellectual-property-policy">read Canonical’s IP policy</a>.
+        </p>
+        <p>
+          If you have any questions about the platform or would like information
+          about our certification program, <a
+          href="/download/contact-us#contact-us" class="p-button--positive
```

Comment:
this should be a simple link, according to design, not a button

Reply:
Oops! Thanks, fixed.